src/Client/Web/Bit.Client.Web.BlazorUI/Components/ProgressIndicator/BitProgressIndicator.razor.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class BitProgressIndicator
+    {
+        private double percentComplete;
+        private bool HasSetPercent;
```

Comment:
please change this variable name to "PercentCompleteHasBeenSet"

Reply:
Renamed HasSetPercent to PercentCompleteHasBeenSet